modules/videopress/shortcode.php

```diff
@@ -150,7 +150,7 @@ function videopress_handle_editor_view_js() {
 
 	add_action( 'admin_print_footer_scripts', 'videopress_editor_view_js_templates' );
 
-	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'jquery' ), false, true );
+	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'media-editor', 'jquery' ), false, true );
```

Comment:
This is still broken for me :cry:, but using you example, rather than blocking the script I was able to work around

```php
wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'media-editor',  'jquery', 'media-element' ), false, true );
```